pkg/loop/internal/pb/contract_reader.proto

```diff
@@ -46,7 +49,10 @@ message UnbindRequest {
 
 // GetLatestValueReply has return arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ContractReader.GetLatestValue].
 message GetLatestValueReply {
-  VersionedBytes ret_val = 1;
+  oneof ret_val {
+    VersionedBytes versioned_bytes = 1;
+    values.Value value = 2;
```

Comment:
I was picturing that it would always send versioned bytes, then the decoder would unmarshal to values.Value if the type is versioned bytes.